src/player_team.ts

```diff
@@ -997,7 +997,22 @@ class Team {
       mults[i].base = pings[i].damage;
     }
 
-    let healing = 0;
+    // Apply poison damage.
+    let poison = 0;
+    for (const c of 'pm') {
+      const attr = COLORS.indexOf(c) as Attribute;
+      for (const combo of comboContainer.combos[c]) {
+        let multiplier = 0;
+        if (c == 'p') {
+          multiplier = 0.2 + (combo.count - 3) * 0.05;
```

Comment:
This can be simplified to (combo.count + 1) * 0.05

Reply:
Agreed. I wrote it this way because it felt like a more intuitively readable implementation of how the formula is usually given (20% for a match-3, 5% for each additional orb in the combo). I'd prefer to keep it in case the formula ever changes, but if you feel strongly about it, I'll change it.

Reply:
The formula I've written is how I think about it: 5% per orb + 1 extra orb per combo.